render/fx_renderer/fx_pass.c

```diff
@@ -374,19 +384,19 @@ void fx_render_pass_add_rounded_rect(struct fx_gles_render_pass *pass,
 
 	struct quad_round_shader *shader = NULL;
 	switch (fx_options->corner_location) {
-	case ALL:
+	case CORNER_LOCATION_ALL:
 		shader = &renderer->shaders.quad_round;
 		break;
-	case TOP_LEFT:
+	case CORNER_LOCATION_TOP_LEFT:
 		shader = &renderer->shaders.quad_round_tl;
 		break;
-	case TOP_RIGHT:
+	case CORNER_LOCATION_TOP_RIGHT:
 		shader = &renderer->shaders.quad_round_tr;
 		break;
-	case BOTTOM_LEFT:
+	case CORNER_LOCATION_BOTTOM_LEFT:
 		shader = &renderer->shaders.quad_round_bl;
 		break;
-	case BOTTOM_RIGHT:
+	case CORNER_LOCATION_BOTTOM_RIGHT:
```

Comment:
I think this breaks upstream -git. We can add this as a follow-up, since we'd also probably want to update the shader anyway. We should handle the individual corner rounding for the rects the same as we do the tex

Reply:
actually let's just update upstream!